src/props.animations.css

```diff
@@ -120,10 +120,24 @@
   100% { opacity: 1; filter: brightness(1) blur(0) }
   };
 }
+[data-theme=dark] {
```

Comment:
nice find 👍🏻

Reply:
Instead of using selectors and media this shoudl be using variables so that you can keep the theme in one place.